tests/functional/centos/7/journal-collocation/scenario.py

```diff
@@ -14,6 +14,7 @@
     },
     'osd0': {
         'username': 'vagrant',
-        'components': [],
+        'components': ['collocated_journals'],
+        'devices': ['/dev/sda', '/dev/sdb'],
```

Comment:
It's a bummer that we have to list devices here and in the group_vars, but I'm not sure how to get around that.